contracts/utils/FuseFixer.sol

```diff
@@ -0,0 +1,182 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import {PCVDeposit} from "../pcv/PCVDeposit.sol";
+import {CTokenFuse, CEtherFuse} from "../external/fuse/CToken.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {CoreRef} from "../refs/CoreRef.sol";
+import {TribeRoles} from "../core/TribeRoles.sol";
+
+/// @title base class for a Compound PCV Deposit
+/// @author Fei Protocol
+contract FuseFixer is PCVDeposit {
```

Comment:
Do we need this contract to be a PCVDeposit? I don't see it added to the CR oracle in the DAO script.

Reply:
It's a TC script, not a DAO script, so we can't add it to the CR oracle anyways.

Either way, assets in this contract are slated for destruction, as it were, so it doesn't make sense to count them in PCV anyway.

It should be a PCV deposit however so that the PCVGuardian can withdraw the remaining funds (what little should remain) to a safe deposit.